### PR TITLE
82555 review approved page

### PR DESCRIPTION
**Pull request by lee-delarm6**
## Summary

- Get one of the new notification appointments pages styled and ready for data input (sample data on the page only)

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/82555

## Testing done

- Visual and build test only

## Screenshots

Screenshot 2024-05-10 at 2 22 11 PM

### CODE REVIEW

**Comment by lee-delarm6** [comment]
Adding feature flag so we can merge this and others into main

Screenshot 2024-05-20 at 3 11 27 PM

**Review by simiadebowale — DISMISSED** [review]
LGTM

File: src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx [hunk 1]
```diff
@@ -48,6 +49,10 @@ export default function AppointmentsPage() {
   const [hasTypeChanged, setHasTypeChanged] = useState(false);
   let [pageTitle] = useState('VA online scheduling');
 
+  const featureAppointmentDetailsRedesign = useSelector(state =>
```
Comment (simiadebowale) [inline src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx:52]: Can we change the name `featureCCDirectScheduling`

Comment (simiadebowale) [reply src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx:52]: Also can we update the selector in src/applications/vaos/referral-appointments/components/ReferralAppLink.jsx to `selectFeatureCCDirectScheduling` .

**Review by simiadebowale — CHANGES_REQUESTED** [review]

**Comment by lee-delarm6** [comment]
@simiadebowale Resolved those notes you've got on there

**Review by simiadebowale — APPROVED** [review]

**Pull request merged**